src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String REGEX = "[^A-Za-z0-9 ]+";
+    private String tempString;
+    private int supply = 0;
+    private int buy = 0;
+    private String result;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        workWithFile.read(fromFileName);
+        workWithFile.calculate();
+        workWithFile.write(toFileName);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(toFileName, report);
```
your program should look like this

Reply:
Please, explain why?I just want to understand)